spec/support/fields_params_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+RSpec.describe FieldsParams do
+  let(:params) do
+    {
+      jobs: "id,name,email",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.